File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Farinha/07.pl
#Programa que imprime a tabuada de um número fornecido pelo utilizador
print "Tabuada a mostrar:\n";
$t = <STDIN>;
$j = $t;
for ($i = 1; $i <= 10; $i++)
{
	$aux = $i * $t;
	print "$i * $t = $aux\n"
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Farinha/07.rb
#Programa que imprime a tabuada de um número fornecido pelo utilizador
puts "Tabuada a mostrar:"
t = gets
j = t.to_i
for i in 1..10
  aux = i.to_i * t.to_i
  puts "#{i} * #{t} = #{aux}"
end